### PR TITLE
Changed text from EN to SK

### PR DESCRIPTION
**Pull request by arti1221**
- Changed language from EN to SK

### CODE REVIEW

File: hanoi.c [hunk 1]
```diff
@@ -3,11 +3,11 @@
 void hanoi(int n, int a, int b)
 {
   if (n == 1) 
-    printf("move stone 1 from %d to %d\n", a, b);
+    printf("Presun opicu 1 z %d na %d\n", a, b);
```
Comment (omaksi) [inline hanoi.c:6]: Tu mal byt tiez kamen :D

File: hanoi.c [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
   else
   {
     hanoi(n - 1, a, n - a);
-    printf("move stone %d from %d to %d\n", n, a, b);
+    printf("Presun kamen %d z %d na %d\n", n, a, b);
```
Comment (omaksi) [inline hanoi.c:10]: Super, tu je to spravne kamen.

**Review by omaksi — CHANGES_REQUESTED** [review]

File: hanoi.c (hunk 1 above)
Comment (M4jo) [reply hanoi.c]: Už je opravené <3

**Review by omaksi — APPROVED** [review]
Dost dobre 👍

**Pull request merged**